### ISSUE TITLE
Some thumbnails for videos are blurred censored.

### ISSUE THREAD

**Issue by shuttittuppitt** [TeamNewPipe/NewPipe#10940]
### Checklist

- [X] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.26.1

### Steps to reproduce the bug

STEPJ TO REPRODUCE THIS BUG

1) Start Newpipe.
2) The bug affects "specific" videos, but for a good example, search for "The godsend 1980".
3) SOME of the results of that search will show a blurred censored thumbnail. The affected thumbnails are only affected in the search results, but if you click on one of them (& don't press the "Play" button), that video shows it's "normal" thumbnail instead.

### Expected behavior

I expect the thumbnails of ALL of the videos that a search shows me to NOT be blurred censored thumbnails.

### Actual behavior

For some searches (but not ALL searches), SOME of the thumbnails of the search results are blurred & censored.

I have Newpipe's "Restricted mode" option UN-checked, so that isn't it. I also use a VPN sometimes, but the affected searches still show this bug. kit only affects some searches, & it's impossible to know which searches will be affected until you perform a search. Also, this bug does NOT happen if I use a BROWSER (example = Chrome), so it does not seem to be because of youtube's website, but rather because of a problem with Newpipe.

### Affected Android/Custom ROM version

Android 11, not rooted

### Affected device model

Samsung galaxy tab A7 (don't know if it affects other devices or not)

**Comment by AudricV** [comment]
> Toggling on age-restricted videos should automatically make the app fetch the uncensored thumbnails.

No, we get the thumbnails from YouTube. @shuttittuppitt You said that you checked with YouTube and it doesn't happen, were you logged-in to an age-verified account? If so, try without being logged-in, I do not get any unblurred thumbnail on my side.

**Comment by shuttittuppitt** [comment]
Sorry about the lateness of my response. I was so busy that I forgot about this bug report until now. Anyway, I checked the problem with youtube's WEBSITE by logging out & then using the same search (green acres), but the WEBSITE still did not blur censor any of the thumbnails. (That makes sense to me, because "Green acres" was a very family friendly TV show. Then some loser cancelled every TV show that had any TREES in them.) Then I tried the same search in Newpipe. I have no idea WHY the bug was happening yesterday, but it does NOT happen anymore. (Yesterday was the ONLY time that it ever happened to me too, & I'm a long time Newpipe user.) I guess that the wifi had something wrong with it yesterday, but that it's fixed now, so I would go ahead & close this bug report as resolved. Still mysterious why it did not happen with youtube's website though (even with me logged out now).
